CarWheel.java

```diff
@@ -0,0 +1,54 @@
+package com.mateakademia.homework.homework3.task32.lamborghinicar;
+
+    public class CarWheel {
+
+        private double tireTag;
+        private static final double TIRE_OLD = 0.0;
+        private static final double TIRE_NEW = 1.0;
+
+        public CarWheel(){
+            tireTag = 1.0;
+        }
+
+        public CarWheel(double tireTag){
+            if (checkTire(tireTag)){
+                this.tireTag = tireTag;
+            } else {
+                System.out.println("It's cann't  create CarWheel");
+            }
+        }
+
+        // method  check  value inside of [0,1]
+        private boolean checkTire(double tireTag) {
```

Comment:
It seems this method has two responsibility check if value inside of [0,1] and print information.
Let's try to use SOLID principle (S- single responsibility)